ractor.rb

```diff
@@ -888,8 +888,22 @@ def self.main?
     }
   end
 
+  ##
+  # :method: require
+  # :args: feature
+  #
+  # A redefinition of Kernel#require, available inside ractor body. In non-main ractor,
+  # it delegates to the main one to do the real +require+.
+  #
+  #    r = Ractor.new {
+  #      require 'json' # it is a call to Ractor#require
```

Comment:
`it is a call to Ractor#require` is misleading.

* There is no `Ractor#require`
* `Ractor#_require` is called, but user doesn't need to know the internal API

So,

```suggestion
  #      require 'json' # require on the main ractor and wait for the result on this ractor.
```

Reply:
I'm not sure it is in RDoc, but doc/ractor.md is better to write?

Reply:
Yeah, my understanding was imperfect (to put it mildly). I adjusted my mental model via a bit of experimenting, to understand it better. These docs aren’t needed, just hiding internal methods with `:nodoc:` is.

I’ll see about updating `ractor.md` a bit later (with proper explanations).

Sorry for the confusion :pray: